connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/webhook/WebhookExecutables.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.runtime.inbound.webhook;
+
+import io.camunda.connector.api.inbound.Activity;
+import io.camunda.connector.api.inbound.Health;
+import io.camunda.connector.api.inbound.InboundConnectorDefinition;
+import io.camunda.connector.api.inbound.Severity;
+import io.camunda.connector.runtime.inbound.executable.RegisteredExecutable;
+import java.util.*;
+
+public class WebhookExecutables {
+
+  private static final String TAG_QUEUEING = "Queueing";
+  private static final String TAG_ACTIVATION = "Activation";
+
+  private final List<RegisteredExecutable.Activated> executables = new ArrayList<>();
+  private final String context;
```

Comment:
What about having a single field for the executable that is currently active? Could simplify the code and list filtering.